steward/src/commands/orchestrator/start.rs

```diff
@@ -118,7 +118,7 @@ impl Runnable for StartCommand {
                 config.ethereum.gas_price_multiplier,
                 config.ethereum.blocks_to_search,
                 config.cosmos.gas_adjustment,
-                self.orchestrator_only,
+                !self.relay,
```

Comment:
It's a little awkward to essentially use a double negative here, and we should implement this change across both gorc and steward. Perhaps we should first make this update to gorc in both the CLI and adjusting the orchestrator to expect a true value here to relay, update steward with that new version, and then change the behavior in steward.

Reply:
Ugochi thought the same thing. I'll update gorc.